src/txn/txn_rollback_to_stable.c

```diff
@@ -1623,6 +1623,20 @@ __rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start
     }
 }
 
+/*
+ * __check_btree_modified --
+ *     Check that the rollback to stable btree is modified or not.
+ */
+static int
+__check_btree_modified(WT_SESSION_IMPL *session, const char *uri, bool *modified)
```

Comment:
Would you mind renaming this to have a rollback_to_stable prefix, like other functions in this file?